TransferList/TransferList.js

```diff
@@ -140,10 +150,11 @@ const TransferListBase = kind({
 	},
 
 	computed: {
-		renderItem: () => ({elements, list, onSelect, selectedItems, ...rest}) => (data) => {	// eslint-disable-line	enact/display-name
+		renderItem: () => ({elements, list, onSelect, selectedItems, showSelection, ...rest}) => (data) => {	// eslint-disable-line	enact/display-name
 			const {index, 'data-index': dataIndex} = data;
 			const element = elements[index];
-			const selected = -1 !== selectedItems.findIndex((pair) => pair.element === element && pair.list === list);
+			const selectedIndex = selectedItems.findIndex((args) => args.element === element && args.list === list) + 1;
+			const selected = selectedIndex !== 0;
```

Comment:
here i wonder why we have too add + 1 on the first line and then compare it with 0 instead of not adding anything and comparing with "-1"